content/docs/04.5-devx/05-app-profile.md

```diff
@@ -41,19 +38,43 @@ To create your App Profile,
 |Description (optional)   | Description of the App Profile, if any | 
 |Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|
 
-3. **Add Service**: Select a service to start the configuration of the App Profile from the available services and configure the service per the use case.
+3. **Add Service**: Select a service to start configuring the App Profile from the available services and configure the service based on your requirements. The following services are provided:
+
+  * [Container Deployment](/devx/app-profile/container-deployment): Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.
```

Comment:
Instead of reffering containers as methods can we refer its definition from here https://www.docker.com/resources/what-container/